### PR TITLE
Hauki 254 page for resource not found

### PR DESCRIPTION
**Pull request by VilleMiekkoja**
image

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -14,7 +14,7 @@ body {
   overflow-y: overlay;
   display: flex;
   flex-direction: column;
-  font-family: var(--font-default);
+  font-family: HelsinkiGrotesk-Regular, Arial, sans-serif;
```
Comment (minevala) [inline src/index.scss:17]: Nice catch 👍 
Could this be in the variables as app-body-text or something?

**Review by minevala — COMMENTED** [review]
Nice job 👍 
Only one tiny comment

File: src/index.scss (hunk 1 above)
Comment (VilleMiekkoja) [reply src/index.scss:17]: - [ ] Put body font-family to its own variable 'app-body-text'

**Review by minevala — APPROVED** [review]
Nice job 👍

**Pull request merged**